proto/ord_schema.proto

```diff
@@ -53,32 +53,54 @@ message ReactionIdentifier {
 }
 
 message ReactionInput {
-  Compound compound = 1;
-  enum ReactionRole {
-    // Unspecified.
-    REACTION_ROLE_UNKNOWN = 0;  
-    // TODO(ccoley): Do we want to use the definition of a reactant aligned
-    // with Reaxys, or say that any species that contributes heavy atoms 
-    // to a desired product is a reactant? This field might be kind of a 
-    // throwaway anyway...
-    REACTANT = 1;
-    REAGENT = 2;
-    SOLVENT = 3;
-    CATALYST = 4;
-  }
-  ReactionRole reaction_role = 2;
-  // Whether this species was intended to be a limiting reactant.
-  bool is_limiting = 3;
+  // Description or name of this pure substance or mixture that
+  // was added to the reaction vessel. This description is 
```

Comment:
I agree with this interpretation that you elaborated on in #27; I think it's a great idea.